app/src/main/java/com/example/abha_create_verify_android/AuthModeActivity.kt

```diff
@@ -81,8 +81,9 @@ class AuthModeActivity : AppCompatActivity() {
     }
 
     private fun moveToDemographic() {
-        val intent = Intent(this, DemographicActivity::class.java)
-        startActivity(intent)
+        Toast.makeText(this, "The selected auth mode is currently unavailable", Toast.LENGTH_LONG).show()
+//        val intent = Intent(this, DemographicActivity::class.java)
```

Comment:
can we remove the comments?

Reply:
its needed in future, that’s y commented it